src/components/Widgets/ReadMeCard/ReadMeCard.tsx

```diff
@@ -123,14 +123,16 @@ const ReadMeCard = (props: Props) => {
       >
         <MarkdownContent
           content={b64DecodeUnicode(value.content).replace(
-            /\[([^\[\]]*)\]\((?!https?:\/\/)(.*?)(\.md)\)/gim,
-            '[$1]' + `(//${hostname}/${owner}/${repo}/blob/${getRepositoryDefaultBranch(
+            /\[([^\[\]]*)\]\((?!https?:\/\/)(.*?)(\.png|\.jpg|\.jpeg|\.gif|\.webp)\)/gim,
+            '[$1]' + `(//${hostname}/${owner}/${repo}/raw/${getRepositoryDefaultBranch(
               value.url
             )}/` + '$2$3)').replace(
-              /\[([^\[\]]*)\]\((?!https?:\/\/)(.*?)(\.png|\.jpg|\.jpeg|\.gif|\.webp)\)/gim,
-              '[$1]' + `(//${hostname}/${owner}/${repo}/raw/${getRepositoryDefaultBranch(
-                value.url
-              )}/` + '$2$3)')}
+            /\[([^\[\]]*)\]\((?!https?:\/\/)docs\/(.*?)(\.md)\)/gim,
+            '[$1](docs/$2/)').replace(
+            /\[([^\[\]]*)\]\((?!https?:\/\/)(.*?)(\.md)\)/gim,
+            '[$1]' + `(//${hostname}/${owner}/${repo}/raw/${getRepositoryDefaultBranch(
```

Comment:
I don't think we want to link to raw files in this case (though it makes sense for images above)

Reply:
Good catch! I've changed the general replace to `blob`